packages/features/ee/workflows/lib/reminders/smsReminderManager.ts

```diff
@@ -85,10 +85,12 @@ export const scheduleSMSReminder = async (
   }
 
   if (message.length > 0 && reminderPhone) {
-    //send SMS when event is booked/cancelled
+    //send SMS when event is booked/cancelled/Reschdeuled
     if (
-      triggerEvent === WorkflowTriggerEvents.NEW_EVENT ||
-      triggerEvent === WorkflowTriggerEvents.EVENT_CANCELLED
+      triggerEvent ===
+      (WorkflowTriggerEvents.NEW_EVENT ||
+        WorkflowTriggerEvents.EVENT_CANCELLED ||
+        WorkflowTriggerEvents.RESCHEDULE_EVENT)
```

Comment:
same here
```suggestion
        workflow.trigger === WorkflowTriggerEvents.BEFORE_EVENT ||
	        workflow.trigger === WorkflowTriggerEvents.NEW_EVENT ||
	        workflow.trigger === WorkflowTriggerEvents.RESCHEDULE_EVENT
```

Reply:
<img width="1105" alt="Screenshot 2022-08-30 at 9 03 52 PM" src="https://user-images.githubusercontent.com/53316345/187479263-836f1929-697c-4734-b969-c2edd0dfed09.png">

@CarinaWolli  It's already updated . i think you are seeing the previous version.